src/TypeScript/EslintBridgeClient/EslintBridgeHttpWrapper.cs

```diff
@@ -55,13 +55,13 @@ public async Task<string> PostAsync(Uri serverEndpoint, object request, Cancella
             var serializedRequest = request == null
                 ? string.Empty
                 : JsonConvert.SerializeObject(request, Formatting.Indented);
-            logger.LogDebug(Resources.INFO_RequestDetails, serverEndpoint, serializedRequest);
```

Comment:
There were a number of messages that were previously INFO that have been downgraded to DEBUG.
For consistency with the other debug messages, I've removed those downgraded messages from the resource file.